policyengine_us/variables/gov/states/ny/tax/income/credits/ny_inflation_refund_credits.py

```diff
@@ -0,0 +1,40 @@
+from policyengine_us.model_api import *
+
+
+class ny_inflation_refund_credits(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New York 2025 inflation refund credits"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.nysenate.gov/legislation/laws/TAX/606#QQQ"
+    defined_for = StateCode.NY
+
+    def formula(tax_unit, period, parameters):
```

Comment:
instead of the `if` statement:
```python
def formula(...):
    return 0

def formula_2025(...):
    # Current code

def formula_2026(...):
    return 0
```
this aligns more with our conventions